From 6ac2c8357089fdce0876dedc197b96dfa72c4a6a Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 19 Feb 2015 09:55:12 +0100
Subject: [PATCH] dracut.sh: call bash with --norc

We don't want any user aliases or functions to be defined in dracut.
---
 dracut.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut.sh b/dracut.sh
index 5e2feba..17bf8f2 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash --norc
 #
 # Generator script for a dracut initramfs
 # Tries to retain some degree of compatibility with the command line
